compiler-rt/test/sanitizer_common/TestCases/Linux/free_aligned_sized.c

```diff
@@ -1,13 +1,25 @@
 // RUN: %clang -std=c23 -O0 %s -o %t && %run %t
-// UNSUPPORTED: asan, hwasan, rtsan, ubsan
+// UNSUPPORTED: asan, hwasan, ubsan
 
 #include <stddef.h>
 #include <stdlib.h>
 
+#if defined(__has_feature) && __has_feature(realtime_sanitizer)
+#  include <sanitizer/rtsan_interface.h>
+#endif
+
+extern void *aligned_alloc(size_t alignment, size_t size);
+
 extern void free_aligned_sized(void *p, size_t alignment, size_t size);
 
 int main() {
+#if defined(__has_feature) && __has_feature(realtime_sanitizer)
+  __rtsan_disable();
```

Comment:
You shouldn't need to disable and re-enable here, be cause `main` isn't marked as nonblocking.

Can you share what happens when you omit these calls?

Reply:
Ah, I was not entirely away of how RTSan worked. So I just assumed it was needed. Removed.